Tests/Pcap++Test/Tests/FilterTests.cpp

```diff
@@ -4,6 +4,7 @@
 #include "EthLayer.h"
 #include "VlanLayer.h"
 #include "IPv4Layer.h"
+#include "IPv6Layer.h"
```

Comment:
are there all new contractors in the test?

Reply:
What does 'new contractors' mean?

Reply:
ops, sorry, I meant the all constructors added in this PR.

Reply:
Should be although I haven't written any specific tests for the `IPAddress` ctors. The old `string` ctors delegate to the `IPAddress` ctors, so covering the string ones should cover the new ones.

With the `IPNetwork` refactor, there is a new test about constructing an `IPFilter` from `IPNetwork` object.